### DISCUSSION TITLE
Not seeing trace view

### DISCUSSION THREAD

**Discussion by korggy** [datalust/seq-tickets#2076] [General]
I just upgraded to Seq 2024.1 but I'm not seeing the trace view in the results. Am I doing something wrong? Is there any tutorial/walkthrough on how to use the new tracing features?
![image](https://github.com/datalust/seq-tickets/assets/1387687/e9d2d4d6-5045-48df-8840-e89f1a3f8698)

**Comment by liammclennan** [comment]
Hi @korggy 

Traces are a different kind of telemetry data. You will need to ingest traces before they appear in the UI. See https://docs.datalust.co/docs/tracing-from-net

**Comment by korggy** [comment]
What should we do if we want to use BOTH Serilog and SerilogTracing? I noticed that everything gets logged twice. Should we remove .WriteTo.Seq altogether and replace it with .WriteTo.SeqTracing?

**Reply by nblumhardt** [reply]
Hi! Yes, `SeqTracing()` records logs, too; it might not be a great name for that method, sorry. We're expecting to improve this at some future point :-)
